client/webserver/site/src/js/app.js

```diff
@@ -511,7 +511,12 @@ export default class Application {
         break
       }
       case 'conn':
-        this.reconnected()
+        // TODO: if the page is presently loaded to markets showing "Connection
+        // to dex server failed" etc. indicating the initial DEX connection was
+        // never established and it is now connected for the first time, this
+        // should reload the markets page, but not for the initial connection.
+        // But we do not want to reload the page for every connect e.g.
+        // connections created by getfee and register.
```

Comment:
`App.reconnected` appears to be unused now.

Reply:
It's still bound to the messagesocket's connect handler `ws.connect(getSocketURI(), this.reconnected)` right?

Reply:
To summarize some chat, `ws.connect(getSocketURI(), this.reconnected)` is for dealing with browser->dexc connection changes where a fairly brute reload is not unwarranted, while a `'conn'` notification from dexc indicates the dexc->dcrdex server connection changed.  So however we end up handling these things,
- We don't want to reload just any page when a dex server connection changes. It's not inappropriate on the markets page, but it does break the register page flow, so this `'conn'` case should be removed.
- For the case where a configured dex server's connection has not yet been established (showing the message on the markets page, the solution from PR #989), to get the markets page loaded, we'd need either a page reload or a lot more info in the `conn` ntfn because we don't have any previous server asset or market config.
- For the browser->dexc reconnect wired by `ws.connect(...)`, that's also a bit excessive to reload the window, but not really a problem.